### PR TITLE
Uploaded a new entry Par0062 for parameter file.

### PR DESCRIPTION
**Pull request by RuishengSu**

### CODE REVIEW

File: models/Par0062/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# The title of the project.
+autoTICI: Automatic Brain Tissue Reperfusion Scoring on 2D DSA Images of Acute Ischemic Stroke Patients
+### Note the three hashtags paragraph captions
```
Comment (ViktorvdValk) [inline models/Par0062/README.md]: Could you replace this template header

File: models/Par0062/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# The title of the project.
```
Comment (ViktorvdValk) [inline models/Par0062/README.md]: And could you replace the template header with the title of the project

**Review by ViktorvdValk — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution to the Model Zoo.
Could you please update the readme and upload to a new folder par0063 (and change the .txt file name), since par0062 was already taken just before you made the PR?

**Comment by RuishengSu** [comment]
Hi Viktor,

Thanks for the kind reminder. I have now updated the pull request accordingly. Could you check if this is OK?

Regards,
Ruisheng


-------- 原始邮件 --------
发件人： Viktor van der Valk <notifications@github.com>
日期： 2020年12月21日周一 中午11:26
收件人： SuperElastix/ElastixModelZoo <ElastixModelZoo@noreply.github.com>
抄送： RuishengSu <ruisheng.su@hotmail.com>, Author <author@noreply.github.com>
主 题： Re: [SuperElastix/ElastixModelZoo] Uploaded a new entry Par0062 for parameter file. (#7)

@ViktorvdValk requested changes on this pull request.

Thanks a lot for your contribution to the Model Zoo.
Could you please update the readme and upload to a new folder par0063 (and change the .txt file name), since par0062 was already taken just before you made the PR?

________________________________

In models/Par0062/README.md<https://github.com/SuperElastix/ElastixModelZoo/pull/7#discussion_r546622371>:

> @@ -0,0 +1,46 @@
+# The title of the project.
+autoTICI: Automatic Brain Tissue Reperfusion Scoring on 2D DSA Images of Acute Ischemic Stroke Patients
+### Note the three hashtags paragraph captions


Could you replace this template header

________________________________

In models/Par0062/README.md<https://github.com/SuperElastix/ElastixModelZoo/pull/7#discussion_r546622540>:

> @@ -0,0 +1,46 @@
+# The title of the project.


And could you replace the template header with the title of the project

―

**Pull request merged**